src/misc/ontime.ts

```diff
@@ -0,0 +1,229 @@
+import { Table } from 'craftjs-plugin/database';
+import { OfflinePlayer, Statistic, Bukkit } from 'org.bukkit';
+import { Player } from 'org.bukkit.entity';
+import { getTable } from '../common/datas/database';
+import { UUID } from 'java.util';
+import { addTranslation, t } from '../common/localization/localization';
+import { color, text } from 'craftjs-plugin/chat';
+import { errorMessage } from '../chat/system';
+import { PlayerQuitEvent } from 'org.bukkit.event.player';
+import { ticksToTime } from '../common/helpers/time';
+import { getOnlinePlayerNames } from '../common/helpers/player';
+
+const TOP_LIST_DEFAULT = 10;
+const TOP_LIST_MAX = 100;
+
+const ontimesDb: Table<string, number> = getTable('ontime-table');
+const ontimes: Map<string, number> = new Map();
```

Comment:
Separate Map is probably unnecessary, because your values don't need to be serialized to JSON. Backing H2 MVStore is unlikely to be more than a few times slower for simple set/get than ES6 Map.